resources/views/app.blade.php

```diff
@@ -9,9 +9,12 @@
 
         <!-- Fonts -->
         <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Nunito:wght@400;600;700&display=swap">
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300&display=swap" rel="stylesheet">
 
         <!-- Styles -->
         <link rel="stylesheet" href="{{ mix('css/app.css') }}">
+        <link rel="stylesheet" href="{{ asset('css/global.css') }}">
```

Comment:
Perhaps in the future we could refactor this into a component since it's the same for app.blade and guest.blade. 
For now this does work so I approve of these changes

Reply:
Could this be made into a separate issue/task for tracking purposes?